photon_installer/isoInstaller.py

```diff
@@ -54,7 +59,13 @@ def __init__(self, options):
                 return
 
         if ks_path:
-            install_config=self._load_ks_config(ks_path)
+            install_config = self._load_ks_config(ks_path)
+
+        # insecure_installation flag added through commandline overrides that of ks_config
+        if k_param:
+            if not install_config:
+                install_config = {}
+            install_config['insecure_installation'] = self.insecure_installation
```

Comment:
can we try initialising  self.insecure_installation = None instead of False
User will set insecure_installation= true or false

if self.insecure_installation  is None: then we know there was no cmdline param set for insecure_installation
Thus, we can remove k_param.

Reply:
yes

Reply:
Done